README.md

```diff
@@ -167,6 +167,9 @@ Options:
   -wt, --with-type <with-type>                               A type to be used for the given enum declaration during binding generation. []
   -wu, --with-using <with-using>                             A using directive to be included for the given remapped declaration name during binding generation. []
   -?, -h, --help                                             Show help and usage information
+
+Wildcards:
```

Comment:
The commands above need to be updated as well. Several of them should say `Supports wildcards.` now.